pact4s/src/test/scala/org/broadinstitute/dsde/workbench/sam/provider/SamProviderSpec.scala

```diff
@@ -157,6 +173,8 @@ class SamProviderSpec
   //
   def requestFilter: ProviderRequest => ProviderRequestFilter = customFilter
 
+  private def reset(): Unit = mockCriticalSubsystemsStatus(true).unsafeRunSync()
```

Comment:
nit - would prefer `reset()` be renamed to something slightly more descriptive

Reply:
Renamed `reset()` to `reInitializeStates()`.